profiles/general.yml

```diff
@@ -8,3 +8,10 @@ bot_mode: manual
 # Starter - used when `bot_mode` is set to `starters`
 # `Treecko`, `Torchic`, `Mudkip`, `Bulbasaur`, `Charmander`, `Squirtle`, `Chikorita`, `Cyndaquil`, `Totodile`
 starter: Mudkip
+
+
+# Select what pokemon to save as a pk file
+# save_all_pokemon overrides the other two
+save_all_pokemon: true    
```

Comment:
nit: I would probably have `save_all_pokemon` as `false` by default, I can't imagine people want 3 million .pk3 files, primary use case would be for shinies/custom catch

Reply:
I'll change it :)